app/src/main/java/org/simple/clinic/home/patients/illustration/HomescreenIllustration.kt

```diff
@@ -0,0 +1,29 @@
+package org.simple.clinic.home.patients.illustration
+
+import androidx.room.Dao
+import androidx.room.Embedded
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import androidx.room.Query
+
+@Entity
+data class HomescreenIllustration(
+    @PrimaryKey
+    val eventId: String,
+
+    val illustrationUrl: String,
+
+    @Embedded(prefix = "from_")
+    val from: DayOfMonth,
+
+    @Embedded(prefix = "to_")
+    val to: DayOfMonth
+) {
+
+  @Dao
+  interface RoomDao {
+
+    @Query("SELECT * FROM HomescreenIllustration")
+    fun illustrations(): List<HomescreenIllustration>
```

Comment:
Since we are not using this now, it's fine. But this should ideally be an `Observable<List<HomescreenIllustration>>`.

Reply:
We won't need an Observable here because it is not something we will be constantly polling. We will only set the illustration once the app is opened and that's it.

Reply:
Makes sense. We'll have to ensure queries run on the IO thread in the controller, though,